common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/Utils.java

```diff
@@ -23,4 +24,22 @@ public static void copyResultFields(MojoFrame mojoFrame, List<Row> outputRows) {
       }
     }
   }
+
+  /**
+   * Sanitize boolean string literal values true / false (case insensitive) into 1 / 0 respectively.
+   * @return sanitized string.
+   */
+  public static String sanitizeBoolean(String value, DataField.DataTypeEnum dataType) {
+    if (
```

Comment:
should it be false is empty?
also, how about `T`/`F`?

Reply:
> should it be false is empty?

We try not to assume how to handle missing values, the responsibility moves down to the MOJO pipeline as the processing of missing values should be consistent with C++ MOJO runtime as well, where we just leave the input to be whatever they were in this case.

> also, how about T/F?

the assumption is when data type is `float` and input is `true / false` case insensitive then we casting into `1 / 0` respectively,  because scoring pipeline and mojo pipeline cast all boolean columns as float type. in my impression, `T / F` is too fishy to make any assumption about the original column, where it could also mean some categorical features.

Reply:
@achraf-mer we're trying to accomdate DAIs behavior. How does DAI handle `T/F`? Are there any other boolean conversions made by DAI?